src/main/java/mate/academy/bookstore/dto/order/AddressRequestDto.java

```diff
@@ -0,0 +1,8 @@
+package mate.academy.bookstore.dto.order;
+
+import jakarta.validation.constraints.NotNull;
+
+public record AddressRequestDto(
+        @NotNull(message = "Address cannot be null")
```

Comment:
@NotBlank is better for check String
because @NotNull can pass empty strings like "" or "   "